### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mroderick**
This PR updates all outdated dependencies

This is ready for review

@cjohansen @mantoni @fatso83

### CODE REVIEW

**Comment by mantoni** [comment]
Nice! That's how it should be, isn't it?

**Comment by mroderick** [comment]
Yep :-)

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ sudo: false
 node_js:
   - "0.10"
   - "0.12"
+  - "4.2"
```
Comment (fatso83) [inline .travis.yml]: I know this is merged already, but maybe we should strive to always have a test for the latest major version (5 at this time)?

Comment (cjohansen) [inline .travis.yml]: +1

**Comment by cjohansen** [comment]
:+1:

**Pull request merged**